graphdatascience/tests/integration/test_database_ops.py

```diff
@@ -32,6 +32,24 @@ def test_switching_db(runner: Neo4jQueryRunner) -> None:
     runner.run_query("DROP DATABASE $dbName", {"dbName": MY_DB_NAME})
 
 
+@pytest.mark.skip_on_aura
+def test_run_query_with_db(runner: Neo4jQueryRunner) -> None:
+    runner.run_query("CREATE (a: Node)")
+
+    default_db_count = runner.run_query("MATCH (n: Node) RETURN COUNT(n) AS c")["c"][0]
+    assert default_db_count == 1
+
+    MY_DB_NAME = "my-db"
+    runner.run_query("CREATE DATABASE $dbName", {"dbName": MY_DB_NAME})
```

Comment:
this is an enterprise feature (https://neo4j.com/docs/cypher-manual/current/databases/#administration-databases-create-database). So make sure we use a Neo4j enterprise edition

Reply:
THis is a problem in other client tests as well. For now we should say that Neo4j enterprise is needed in our dev docs?

Reply:
the alternative would be to mark tests with another "neo4j-enterprise" marker. Seems a bit complicated? Maybe we keep it like this until someone complains and do the work to have a "neo4j CE"-mode later?

Reply:
agreed. We also learned yesterday that Neo4j desktop includes neo4j EE by default